spring-cloud-dataflow-server-kubernetes/pom.xml

```diff
@@ -1,66 +1,107 @@
+			<comments>
+				Copyright 2014-2016 the original author or authors.
+
+				Licensed under the Apache License, Version 2.0 (the "License");
+				you may not use this file except in compliance with the License.
+				You may obtain a copy of the License at
+
+				http://www.apache.org/licenses/LICENSE-2.0
+
+				Unless required by applicable law or agreed to in writing, software
+				distributed under the License is distributed on an "AS IS" BASIS,
+				WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+				implied.
+
+				See the License for the specific language governing permissions and
+				limitations under the License.
+			</comments>
+		</license>
+	</licenses>
+
+	<scm>
+		<url>https://github.com/spring-cloud/spring-cloud-dataflow-server-kubernetes</url>
+		<connection>scm:git:git://github.com/spring-cloud/spring-cloud-dataflow-server-kubernetes.git</connection>
+		<developerConnection>scm:git:ssh://git@github.com/spring-cloud/spring-cloud-dataflow-server-kubernetes.git</developerConnection>
+		<tag>HEAD</tag>
+	</scm>
+
 	<properties>
-		<start-class>org.springframework.cloud.dataflow.server.kubernetes.KubernetesDataFlowServer</start-class>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+		<java.version>1.8</java.version>
+		<spring-cloud-task.version>1.1.0.RC1</spring-cloud-task.version>
+		<spring-cloud.version>Camden.SR2</spring-cloud.version>
 	</properties>
+
 	<dependencies>
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-starter-dataflow-server-kubernetes</artifactId>
 			<version>1.1.0.BUILD-SNAPSHOT</version>
 		</dependency>
+
 		<dependency>
-			<groupId>org.mariadb.jdbc</groupId>
-			<artifactId>mariadb-java-client</artifactId>
-			<scope>runtime</scope>
-		</dependency>
-		<dependency>
-			<groupId>org.postgresql</groupId>
-			<artifactId>postgresql</artifactId>
-			<scope>runtime</scope>
-		</dependency>
-		<!-- Fabric8 spring-cloud-kubernetes dependencies -->
-		<dependency>
-			<groupId>io.fabric8</groupId>
-			<artifactId>spring-cloud-starter-kubernetes</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>io.fabric8</groupId>
-			<artifactId>spring-cloud-kubernetes-archaius</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>com.netflix.archaius</groupId>
-			<artifactId>archaius-core</artifactId>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-test</artifactId>
+			<scope>test</scope>
 		</dependency>
 	</dependencies>
+
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-dataflow-server-kubernetes-dependencies</artifactId>
+				<version>1.1.0.BUILD-SNAPSHOT</version>
+				<type>pom</type>
+				<scope>import</scope>
+			</dependency>
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
```

Comment:
I see. But initializr does that for local because of [this](https://github.com/ericbottard/initializr/blob/AddDataFlowK8S/initializr-service/src/main/resources/application.yml#L45). Seems more sensible to then depend on some "core" SCDF bom